### PR TITLE
✨  Add swap preview modal

### PR DESCRIPTION
**Pull request by git-marcopitra**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206265310509608

### CODE REVIEW

File: views/swap/index.tsx [hunk 1]
```diff
@@ -74,14 +90,15 @@ const Swap: FC = () => {
...
-              bg="container"
-              opacity="0.4"
-              color="onSurface"
               px="l"
               py="s"
-              borderRadius="xs"
-              variant="tonal"
               fontSize="s"
+              opacity="0.4"
+              bg="container"
```
Comment (MarioBatalha) [inline views/swap/index.tsx:97]: Add condition to change  button text color and bg when 'from' and 'to' coins exist.

File: views/swap/swap-preview-modal/index.tsx [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+            px="xl"
+            display="flex"
+            flexDirection="column"
+            justifyContent="start"
+          >
+            <Typography size="medium" variant="label" mb="xs">
+              FROM
+            </Typography>
+            <Box
+              bg="#EDEDF1"
```
Comment (MarioBatalha) [inline views/swap/swap-preview-modal/index.tsx:65]: Use the theme value

File: views/swap/swap-preview-modal/index.tsx [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+              pt="2xs"
+              display="flex"
+              flexDirection="column"
+              justifyContent="start"
+            >
+              <Typography size="medium" variant="label" mb="xs">
+                TO (ESTIMATED)
+              </Typography>
+              <Box
+                bg="#EDEDF1"
```
Comment (MarioBatalha) [inline views/swap/swap-preview-modal/index.tsx:119]: Here too

**Review by MarioBatalha — CHANGES_REQUESTED** [review]

File: views/airdrop/airdrop-form/airdrop-preview-modal.tsx [hunk 4]
```diff
@@ -39,95 +38,93 @@ const AirdropPreviewModal: FC<AirdropPreviewModalProps> = ({
...
+          flexDirection="column"
+          justifyContent="start"
+        >
+          <Typography size="medium" variant="label" mb="xs">
+            You will send
+          </Typography>
           <Box
-            p="xl"
+            bg="#EDEDF1"
+            p="0.75rem"
```
Comment (KipandaJr) [inline views/airdrop/airdrop-form/airdrop-preview-modal.tsx:86]: get this value to the theme

File: views/airdrop/airdrop-form/airdrop-preview-modal.tsx [hunk 5]
```diff
@@ -39,95 +38,93 @@ const AirdropPreviewModal: FC<AirdropPreviewModalProps> = ({
...
+          display="flex"
+          flexDirection="column"
+          justifyContent="start"
+        >
+          <Typography size="medium" variant="label" mb="xs">
+            You will send
+          </Typography>
           <Box
-            p="xl"
+            bg="#EDEDF1"
```
Comment (KipandaJr) [inline views/airdrop/airdrop-form/airdrop-preview-modal.tsx:85]: we have this color on the theme

File: views/swap/swap-preview-modal/index.tsx [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+            display="flex"
+            flexDirection="column"
+            justifyContent="start"
+          >
+            <Typography size="medium" variant="label" mb="xs">
+              FROM
+            </Typography>
+            <Box
+              bg="#EDEDF1"
+              p="0.75rem"
```
Comment (KipandaJr) [inline views/swap/swap-preview-modal/index.tsx:66]: Use the theme value

File: views/swap/swap-preview-modal/index.tsx [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+              display="flex"
+              flexDirection="column"
+              justifyContent="start"
+            >
+              <Typography size="medium" variant="label" mb="xs">
+                TO (ESTIMATED)
+              </Typography>
+              <Box
+                bg="#EDEDF1"
+                p="0.75rem"
```
Comment (KipandaJr) [inline views/swap/swap-preview-modal/index.tsx:123]: Use the theme value

**Review by KipandaJr — CHANGES_REQUESTED** [review]

**Pull request merged**